acp/ideas_module.php

```diff
@@ -150,10 +207,20 @@ public function main($id, $mode)
 			}
 		}
 
-		if ($submit)
+		// Submit relevant log entries and output success message
+		if ($submit || $submit_forum_setup)
 		{
-			// Add option settings change action to the admin log
-			$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_SETTINGS_LOG');
+			if ($submit)
+			{
+				// Add option settings change action to the admin log
+				$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_SETTINGS_LOG');
+			}
+
+			if ($submit_forum_setup)
+			{
+				// Add forum setup action to the admin log
+				$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, 'ACP_PHPBB_IDEAS_FORUM_SETUP_LOG');
+			}
```

Comment:
Also, if we really want to be cool, we can set up the log and trigger_error messages so we can easily display a unique message to the user:
```php
$message = ($submit_forum_setup) ? 'FORUM_SETUP' : 'SETTINGS';

$this->log->add('admin', $this->user->data['user_id'], $this->user->ip, "ACP_PHPBB_IDEAS_{$message}_LOG";

trigger_error($this->language->lang("ACP_PHPBB_IDEAS_{$message}_CHANGED") . adm_back_link($this->u_action));
```